### PR TITLE
ERC865

### PR DESCRIPTION
**Pull request by wagyu298**
Using ERC865 pull request of zeppelin-solidity.
This pull request changed task runner from npm to yarn.

### CODE REVIEW

**Comment by ms76** [comment]
@wagyu298 Can you please check what I commented?

File: test/ExampleCoinMock.js [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+        }
+      );
+
+      const allowance = await instance.allowance.call(buyer, spender);
+      assert.ok(allowance.equals(new BigNumber(amount).add(increase)));
+    });
+
+    it('should reject with invalid signature', async () => {
+      const nonce = web3.eth.getTransactionCount(buyer);
+      const hashedTx = await instance.approvePreSignedHashing.call(
```
[the remark this answers is not in the document]
Comment (wagyu298) [reply test/ExampleCoinMock.js]: I fixed the suggestion but GitHub does not close comments. It seems to like GitHub problem.

File: test/ExampleCoinMock.js [hunk 2]
```diff
@@ -0,0 +1,640 @@
...
+        }
+      );
+
+      const allowance = await instance.allowance.call(buyer, spender);
+      assert.ok(allowance.equals(new BigNumber(amount).sub(decrease)));
+    });
+
+    it('should reject with invalid signature', async () => {
+      const nonce = web3.eth.getTransactionCount(buyer);
+      const hashedTx = await instance.approvePreSignedHashing.call(
```
[the remark this answers is not in the document]
Comment (wagyu298) [reply test/ExampleCoinMock.js]: It is same GitHub problem.

**Comment by wagyu298** [comment]
@ms76 Thank you for the coding reviews.
I fix your suggestions and I added zero fee test cases for transferPreSigned and transferFromPreSigned.

**Comment by ms76** [comment]
@wagyu298 LGTM 👍

**Pull request merged**